skills/shepherd/SKILL.md

```
@@ -90,6 +90,33 @@ For each comment thread, check if it has been **replied to** (has child comments
 - **PR #123** — Graphite: `resolveEvalsDir` should use injected config (eval-run.ts:14)
 ```
 
+### Gate Event Emission
+
+After checking CI status for each PR, emit `gate.executed` events to the event store for quality tracking:
+
+For each CI check result observed:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)